catalog-info.yaml

```diff
@@ -47,7 +47,7 @@ metadata:
 spec:
   # Although not officially supported, we can use this type. The spec will be shown as source.
   type: crd
-  owner: "group:default/tachyon"
+  owner: group:default/sda_se_open_industry_solutions
```

Comment:
There is another entity with [owner in line 22](https://github.com/SDA-SE/mongodb-operator/pull/589/files#diff-47e4499e7f2830df90da07ca1636ba625c6a700eedc9671dfc2f7ca69fed42feL22).

Reply:
changed it there too